src/main/java/net/snowflake/ingest/utils/HttpUtil.java

```diff
@@ -283,18 +290,15 @@ static HttpRequestRetryHandler getHttpRequestRetryHandler() {
         LOGGER.info("Max retry exceeded for requestURI:{}", requestURI);
         return false;
       }
-      if (exception instanceof NoHttpResponseException
-          || exception instanceof javax.net.ssl.SSLException) {
-        LOGGER.info(
-            "Retrying request which caused {} with " + "URI:{}, retryCount:{} and maxRetryCount:{}",
-            exception.getClass().getName(),
-            requestURI,
-            executionCount,
-            MAX_RETRIES);
-        return true;
-      }
-      LOGGER.info("No retry for URI:{} with exception {}", requestURI, exception.toString());
-      return false;
+
+      LOGGER.info(
```

Comment:
Should we include the message as well?

Reply:
We are logging `exception.toString()`, which includes both exception class name and message